sql_generators/mobile_kpi_support_metrics/templates/attribution_clients.query.sql

```diff
@@ -0,0 +1,225 @@
+{{ header }}
+WITH new_profiles AS (
+  SELECT
+    submission_date,
+    client_id,
+    sample_id,
+  FROM `{{ project_id }}.{{ dataset }}.active_users`
```

Comment:
is there a reason why we are reading from active_users?
While the data is there, it doesn't seem we are leveraging anything specific to active_users in this query, and it's really hard to trace back the new profile logic from here.
Also, this is a table that has all clients active in the last 28 days, so it's unnecessarily large for this query

Reply:
In this case, I think we could also change this to the baseline_clients_first_seen view to act as the source.